Calculator/Calculator/Grid.xaml.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace Calculator
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+	public partial class Grid : ContentPage
+	{
+        Processor processor = new Processor();
+   		public Grid ()
+		{
+			InitializeComponent ();
+        }
```

Comment:
please, check, what's going on with spacing here. and configure your IDE to insert spaces instead of tabs